src/cashlib/libsecp256k1-config.h

```diff
@@ -0,0 +1,162 @@
+/* Manually configured for Android builds  */
+
+#ifndef LIBSECP256K1_CONFIG_H
+
+#define LIBSECP256K1_CONFIG_H
+
+#define SECP256K1_INLINE inline
+
+/* Define if building universal (internal helper macro) */
+/* #undef AC_APPLE_UNIVERSAL_BUILD */
+
+/* Define this symbol to compile out all VERIFY code */
+/* #undef COVERAGE */
+
+/* Define this symbol to enable the ECDH module */
+/* #undef ENABLE_MODULE_ECDH */
+
```

Comment:
since it is a new file why keeping around all those commented LOC?

Reply:
for documentation.  they describe the options we might choose in the future